src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  font-family: 'Roboto';
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+.container {
```

Comment:
Same thing as before, there should be an empty line between all css styles, just like on lines 15, 11, 5.

In real projects, its really important to keep code style consistent